tests/kernel/fatal/src/main.c

```diff
@@ -36,7 +37,7 @@ static volatile int crash_reason;
 #ifndef CONFIG_ARM
 FUNC_NORETURN
 #endif
-void _SysFatalErrorHandler(unsigned int reason, const NANO_ESF *pEsf)
+void _SysFatalErrorHandler(unsigned int reason, const NANO_ESF * pEsf)
```

Comment:
why was a space inserted here, that's not correct style

Reply:
Checkpatch throws following error if I don't do this:

[pv@pvara-mobl1 new_pr]$ ./scripts/checkpatch.pl -f tests/kernel/fatal/src/main.c 
tests/kernel/fatal/src/main.c:40: ERROR:SPACING: need consistent spacing around '*' (ctx:WxV)

Reply:
leave it as is, there is another way to fix this. (see #4714)

Reply:
@nashif Done